MTR.pm

```diff
@@ -12,8 +12,6 @@
 =head1 SYNOPSIS
 
   mv MTR.pm ~/.vep/Plugins
-  curl -O ftp://mtr-viewer.mdhs.unimelb.edu.au/pub/mtrflatfile_1.0.txt.gz
```

Comment:
Are version 1 files not supported anymore?

Reply:
Version 1 files are still supported but the link does not work anymore. They have changed how it is stored and how it can be downloaded.

Reply:
Also the version2 files are the updated version as version 1 is Ensembl v75 transcript sets while version 2 is Ensembl v95 transcript set. Also version 2 is calculated across over 220,000 exomes using gnomAD, Biobank UK and DiscovEHR while version 1 is calculated across gnomAD only.